### ISSUE TITLE
child_process does not start in windows

### ISSUE THREAD

**Issue by chitabiswal4** [spicecoder/QnAGate_sequence#3]
While executing the sh command it throws some error.

**Comment by spicecoder** [comment]
Yes,looks like we have to follow the spawn method on windows side :https://singhak.in/run-a-bat-file-from-node-js-execute-child-process-from-node-js/
